theme/theme.scss

```diff
@@ -37,4 +37,8 @@
 
 	// Center the image if its width is smaller than content's width.
 	margin: 0 auto;
+
+	// Make sure the image never exceeds the size of the parent container (#67).
+	max-width: 100%;
+	max-height: 100%;
```

Comment:
Are you sure we need `max-height`? For me, there is nothing wrong if the long image is longer then editors editable area. Also, it means that a long image will be resized if editable have dynamic height, isn't it?